docs/AUTH0_SETUP_GUIDE.md

```
@@ -0,0 +1,320 @@
+
+1. In Auth0 Dashboard, go to **Applications** → **Applications**
+2. Click **+ Create Application**
+3. Choose **Regular Web Application**
+4. Name: `Bocchi The Map`
+5. Click **Create**
+
+### 3. Configure Application Settings
+
+In your application's **Settings** tab:
+
+#### Basic Information
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)